gems/actionpack/6.0/actioncontroller.rbs

```diff
@@ -172,3 +174,275 @@ module ActionController
+  module Helpers
+    extend ActiveSupport::Concern
+
+    class MissingHelperError < LoadError
+      def initialize: (untyped error, untyped path) -> untyped
+    end
+
+    module ClassMethods
+      # When a class is inherited, wrap its helper module in a new module.
+      # This ensures that the parent class's module can be changed
+      # independently of the child class's.
+      def inherited: (untyped klass) -> untyped
+
+      # Declare a controller method as a helper. For example, the following
+      # makes the +current_user+ and +logged_in?+ controller methods available
+      # to the view:
+      #   class ApplicationController < ActionController::Base
+      #     helper_method :current_user, :logged_in?
+      #
+      #     def current_user
+      #       @current_user ||= User.find_by(id: session[:user])
+      #     end
+      #
+      #     def logged_in?
+      #       current_user != nil
+      #     end
+      #   end
+      #
+      # In a view:
+      #  <% if logged_in? -%>Welcome, <%= current_user.name %><% end -%>
+      #
+      # ==== Parameters
+      # * <tt>method[, method]</tt> - A name or names of a method on the controller
+      #   to be made available on the view.
+      def helper_method: (*untyped meths) -> untyped
+
+      # The +helper+ class method can take a series of helper module names, a block, or both.
+      #
+      # ==== Options
+      # * <tt>*args</tt> - Module, Symbol, String
+      # * <tt>block</tt> - A block defining helper methods
+      #
+      # When the argument is a module it will be included directly in the template class.
+      #   helper FooHelper # => includes FooHelper
+      #
+      # When the argument is a string or symbol, the method will provide the "_helper" suffix, require the file
+      # and include the module in the template class. The second form illustrates how to include custom helpers
+      # when working with namespaced controllers, or other cases where the file containing the helper definition is not
+      # in one of Rails' standard load paths:
+      #   helper :foo             # => requires 'foo_helper' and includes FooHelper
+      #   helper 'resources/foo'  # => requires 'resources/foo_helper' and includes Resources::FooHelper
+      #
+      # Additionally, the +helper+ class method can receive and evaluate a block, making the methods defined available
+      # to the template.
+      #
+      #   # One line
+      #   helper { def hello() "Hello, world!" end }
+      #
+      #   # Multi-line
+      #   helper do
+      #     def foo(bar)
+      #       "#{bar} is the very best"
+      #     end
+      #   end
+      #
+      # Finally, all the above styles can be mixed together, and the +helper+ method can be invoked with a mix of
+      # +symbols+, +strings+, +modules+ and blocks.
+      #
+      #   helper(:three, BlindHelper) { def mice() 'mice' end }
+      #
+      def helper: (*untyped args) ?{ () -> untyped } -> untyped
```

Comment:
Could you move the methods only what you changed, please? If my understanding is correct, only `ActionController::Helpers::ClassMethods#helper` and `AbstractController::Caching::Fragments::ClassMethods#fragment_cache_key` should be moved and modified.

Reply:
Sorry 🙏 
I wasn't sure how far I should move the auto-generated code, including comments, so I made https://github.com/sanfrecce-osaka/gem_rbs_collection/commit/d7a4fb518abd0c258ea37b46d796bdcc31ba7e1a and https://github.com/sanfrecce-osaka/gem_rbs_collection/commit/08b92b9e5ab942637942e3736972492ad9f5bff0.
I fixed in [939ac11](https://github.com/ruby/gem_rbs_collection/pull/715/commits/939ac1144fa8335cb2233c9b434da784ec2c259f) based on your feedback.